pxr/imaging/plugin/hdRpr/rprApi.cpp

```diff
@@ -3287,31 +3322,45 @@ Don't show this message again?
         try {
             if (!aov) {
                 HdRprApiAov* newAov = nullptr;
+                HdRprApiAovBuilder builder = HdRprApiAov::Builder()
+                    .WithType(rpr::Aov(aovDesc.id))
+                    .WithFormat(format)
+                    .WithSize(width, height)
+                    .WithRprContext(m_rprContext.get())
+                    .WithRprContextMetadata(&m_rprContextMetadata)
+                    .WithRifContext(m_rifContext.get())
+                    .WithRenderResolution(GetRenderResolution());
```

Comment:
So Effectively all these parameters are mandatory. What do you think about making them so Actually?

Reply:
Made a check in Build() for all mandatory parameters